### PR TITLE
feat(sentry): Create 2nd Sentry client for frontend http requests

### PR DESCRIPTION
**Pull request by billyvg**
This was creating a lot of noise, and while I believe has some value, I
think the noise outweighs its value. Lets move this to a separate project
and maybe in the future we can evaluate if we can have APM transactions
solve any needs.

### CODE REVIEW

**Review by markstory — COMMENTED** [review]
Looks good to me, but I'll leave the ✅ up to Matt or Armin. I think APM can cover some of this tracking. However we don't currently index the span details and thus you can't search on it :cry:

**Comment by billyvg** [comment]
I think I was the only one pushing to keep these in `sentry` @markstory - I just want matt to be aware since this will affect our on-prem users.

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -919,6 +919,8 @@ def create_partitioned_queues(name):
 # DSN for the frontend to use explicitly, which takes priority
 # over SENTRY_FRONTEND_PROJECT or SENTRY_PROJECT
 SENTRY_FRONTEND_DSN = None
+# DSN for tracking all client HTTP requests (which can be noisy) [experimental]
+SENTRY_FRONTEND_REQUESTS_DSN = None
```
Comment (markstory) [inline src/sentry/conf/server.py]: Do we have a getsentry pull to define this?

Comment (billyvg) [reply src/sentry/conf/server.py]: Good call, added here, https://github.com/getsentry/getsentry/pull/3623

**Review by markstory — APPROVED** [review]

**Pull request merged**